src/sdk.js

```diff
@@ -1,57 +1,111 @@
 window.Smartcar = (function(window) {
   'use strict';
+  // accesss denied error class
+  class AccessDenied extends Error { // eslint-disable-line no-restricted-syntax
+    constructor(message) {
+      super(message);
+      this.name = 'AccessDenied';
+    }
+  }
 
   /**
    * Initializes Smartcar class
    *
    * @param {Object} options - the SDK configuration object
    * @param {String} options.clientId - the applications' client id
-   * @param {String} options.redirectUri - the registered redirect uri of the application
+   * @param {String} options.redirectUri - the registered redirect uri of the
+   * application
    * @param {String[]} [options.scope] - requested permission scopes
-   * @param {Function} [options.onComplete] - called upon completion of the Auth flow
-   * @param {Boolean} [options.development=false] - launch Smartcar auth in development mode
-   * to enable the mock vehicle brand
+   * @param {Function} [options.onComplete] - called on completion of auth flow
+   * @param {Boolean} [options.development=false] - launch Smartcar auth in
+   * development mode to enable the mock vehicle brand
    * @constructor
    */
   function Smartcar(options) {
+    // allow only one instance of Smartcar
+    if (Smartcar._hasBeenInstantiated) {
+      throw new Error('Smartcar has already been instantiated in the window.' +
+        ' Only one instance of Smartcar can be defined. See' +
+        ' https://github.com/smartcar/javascript-sdk for more information');
```

Comment:
should we point to a specific part of the readme in this link?

Reply:
removed reference as there doesn't seem to be a relevant section in the README